components/alerts/alert-mod.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Alerts</title>
+    <link rel="stylesheet" href="../main-style.css" />
+    <link rel="stylesheet" href="../theme.css" />
```

Comment:
you import all css into a main css file and directly give that reference

Reply:
Noted. This will be very helpful to carry all those individual CSS files. Thanks.